meson/meson.build

```diff
@@ -44,22 +44,27 @@ subproject(
   default_options : [
     'grep=false',
     'test=false',
+    'jit_sealloc=disabled',# arm64 darwin
   ],
 )
-subproject(
-  'glib',
-  default_options : [
-    'nls=disabled',
-  ],
-)
+if host_machine.system() != 'linux'
+  subproject(
+    'glib',
```

Comment:
When loading the binaries on linux, I ran into fatal crashes seemingly because of glib(c) clashes. My logic was that most versions of linux will have glib anyways

Reply:
Sorry to be clear, this was when loading any recent javafx modules and then trying to load openslide, via system.load or native.load, I can provide a reproducible example if needed, but I found that removing it fixed the issue without any apparent drawback

Reply:
I think we should try to include all dependencies that aren't glibc, so yeah, a reproducer would be helpful.